src/plugins/csp_configuration_provider/server/provider.test.ts

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SearchResponse } from '@opensearch-project/opensearch/api/types';
+import {
+  OpenSearchClientMock,
+  opensearchClientMock,
+} from '../../../core/server/opensearch/client/mocks';
+import { OpenSearchCspClient } from './provider';
+
+const INDEX_NAME = 'test_index';
+const INDEX_DOCUMENT_NAME = 'csp_doc';
+const ERROR_MESSAGE = 'Service unavailable';
+
+describe('Provider', () => {
```

Comment:
Could we be more descriptive about test suite name to help locate code when test fail

Reply:
will add